metricbeat/module/ceph/docker-compose.yml

```diff
@@ -13,11 +13,11 @@ services:
       - 8003
       - 8080
   ceph-api:
-    image: docker.elastic.co/integrations-ci/beats-ceph:master-6373c6a-jewel-centos-7-x86_64-1
+    image: docker.elastic.co/integrations-ci/beats-ceph:${CEPH_VERSION:-master-97985eb-nautilus-centos-7-x86_64}-1
```

Comment:
I don't think we can use the same versions in both cases, if I remember well, we kept this here to keep some tests specifically running against some specific older version.

I was wondering how is it possible then that tests pass after the changes in this PR, and I have found that there isn't actually anything using at the moment with the `ceph-api` service. It is not used in the python system tests, and go integration tests are skipped as flaky: https://github.com/elastic/beats/issues/22612

Changes here probably need to be deeper, and maybe not only on the testing side. I think that we need to fix the integration tests and check what system tests are actually testing. Maybe at the end we can use a single `ceph` service, but we need to make E2E and go integration tests smarter so they know what metricsets they should be using for each versions. This is what the system tests seem to be doing.

Or maybe we can refactor the metricsets so they detect the version and do nothing if they implement collection methods that are not supported. This way we could enable all metricsets with any version. But I'd say that changes in this module have low priority at the moment.